nuheat/thermostat.py

```diff
@@ -1,3 +1,4 @@
+from datetime import datetime, timezone, timedelta, time
```

Comment:
i think `timezone` was added in python 3.2... which means this would break support for python 2.x.

this might be ok since python 2.7 has passed it's sunset date tho. i'll just have to make some other changes in the codebase to sunset 2.7 support for this package.

Reply:
Sorry for the trouble here.  I think we can put this one into the category of things that non-Python-experts do.  My particular use case is for Home Assistant, so Python 2.7 is long-since dropped there.  I'm not sure what other uses you need python-nuheat to support, though.

It looks like pytz may support Python 2.7, so if that would be a suitable replacement, we may be able to preserve 2.7 support by switching things to use that.

Reply:
🤔 yea, to preserve 2.7, we will likely need to move to `pytz`.

however, homeassistant is the main reason that i wrote this package and if they're on 3, that's what's most important.

python2 has been sunset for almost 3 years now, so i'll just convert this to a python3+ package.